documentation/docs/client/searching_messages.md

```diff
@@ -0,0 +1,51 @@
+---
+id: clientSearchingMessages
+title: Searching Messages
+sidebar_position: 20
+---
+
+Message search is built-in to the chat API. You can enable and/or disable the search indexing on a per-channel type. 
```

Comment:
`You can enable and/or disable the search indexing on a per-channel type.` - is this documented somewhere in the CMS? If yes, maybe it would be worth it to add link

Reply:
Searching for a page where  "search indexing" is described...